client/src/plus/app.tsx

```diff
@@ -1,426 +1,298 @@
 import React from "react";
```

Comment:
I'm not sure where to place this comment but it bothers me that the header fonts become different. 
It's not a big deal but it "looks buggy". 
<img width="1497" alt="Screen Shot 2021-07-22 at 1 22 10 PM" src="https://user-images.githubusercontent.com/26739/126681955-2185fccc-b384-4d84-82de-9949a4530790.png">

Perhaps it's intentional. Perhaps it's deliberate. It still feels a big "huh??".

Reply:
Yup, it is because we are overriding some fonts. It is going to be a total hack to address it so, I reckon it is ok for the duration of the experiment and only affects these pages.